content/embeds/tls-configuration-procedure.md

```diff
@@ -58,22 +66,24 @@ To enable TLS for Replica Of and client communication on the source database:
     The certificates of the clients that connect to the database.
```

Comment:
Just a suggestion to fix this sentence a bit. Or maybe we could present the two required types of certificates (syncer certificates and client certificates) as bullet points instead.
```suggestion
    You also need to add the certificates of the clients that connect to the database.
```

Reply:
Fixed.